done/done.py

```diff
@@ -183,3 +201,98 @@ def max_score(self):
         """The maximum raw score of our problem.
         """
         return 1
+
+    # ScorableXBlockMixin methods
+    def set_score(self, score):
+        """
+        Sets the score on this block.
```

Comment:
Title of the docstring should be separated from the body of the docstring by one empty line